.travis.yml

```diff
@@ -1,23 +0,0 @@
-language: node_js
-node_js:
-  - 8
-install:
-  - npm install -g gulp
-  - npm install -g codecov
-  - ./setup.sh
-script:
-  - gulp lint --travis
-  - npm test
-  - codecov
-before_deploy:
-  - cp test/index.html dist/index.html
-  - cp test/test-runner.js dist/test-runner.js
-  - cp -r test/sample_data/ dist/sample_data/
-  - cp -r test/static/ dist/static/
-deploy:
-  provider: pages
-  local_dir: dist
-  skip_cleanup: true
-  github_token: $GITHUB_GHPAGES_TOKEN
-  on:
-    branch: master
```

Comment:
Does this need to be migrated to Github Actions? If so, how?

Reply:
The background is that it [keeps the gh-pages branch in sync](https://github.com/cfpb/cfpb-chart-builder/pull/67), so seems like we'd use [this action](https://github.com/marketplace/actions/deploy-to-github-pages)

Reply:
Oh hmm, I wonder if I should leave it off for now anyway, considering https://cfpb.github.io/cfpb-chart-builder/ is broken (see https://github.com/cfpb/cfpb-chart-builder/issues/113). I can leave a note in 113 to add it to GitHub Actions.

Reply:
I'm okay with that, in which case :+1: overall